README.md

```diff
@@ -1,75 +1,62 @@
 
-[Redux](https://github.com/rackt/redux) is cool.
-[react-router](https://github.com/rackt/react-router) is neat. The
-problem is that react-router manages an important piece
-of your application state: the URL. If you are using redux, you want
-your app state to fully represent your UI; if you snapshotted the app
-state, you should be able to load it up later and see the same thing.
+react-router does a great job of mapping the current URL to a component tree, and continually does so with any URL changes. This is very useful, but we really want to store this state in redux as well.
 
-react-router automatically maps the current URL to a path down your
-component tree, and continually does so with any URL changes. This is
-very useful, but we really want to store this state in redux as well.
+The entire state that we are interested in boils down to one thing: the URL. This is an extremely simple library that just puts the URL in redux state and keeps it in sync with any react-router changes. Additionally, you can change the URL via redux and react-router will change accordingly.
 
-The entire state that we are interested in boils down to one thing:
-the URL. This is an extremely simple library that just puts the URL in
-redux state and keeps it in sync with any react-router changes.
-Additionally, you can change the URL via redux and react-router will
-change accordingly.
+##### _What about redux-router?_
 
-```js
-npm install redux-simple-router
-```
+[redux-router](https://github.com/rackt/redux-router) is another project which solves the same problem. However, it's far more complex. Take a quick look at [the code for this library](https://github.com/jlongster/redux-simple-router/blob/master/src/index.js)—it's extremely minimal. redux-router is much bigger and more complex.
+
+That said, redux-router is a fine project and has features this doesn't provide. Use it if you like it better.
+
+**Compared with redux-router:**
 
-## Isn't there already redux-router?
+* Much smaller and simpler. You don't need to learn another library on top of everything else.
+* We encourage direct access of react-router APIs. Need server-side rendering, or something else advanced? Just read react-router's docs.
+* We only store current URL and state, whereas redux-router stores the entire location object from react-router.
 
-[redux-router](https://github.com/rackt/redux-router) is another
-project which solves the same problem. However, it's far more complex.
-Just look at this code: the whole thing is only 68 lines of JS.
-redux-router is much bigger and more complex.
+This library only depends on the `history.listen` function from react-router and the `store.getState` and `store.subscribe` functions from redux. It should be very future-proof with any versions of either libraries.
```

Comment:
:+1: Hopefully @jlongster has got some time to check it out soon so we get a 1.0 out. I have a feeling he's [quite busy](https://wiki.mozilla.org/Coincidental_work_weeks/2015_Orlando) this week.